src/common/axios-config.js

```diff
@@ -3,8 +3,16 @@ import { refreshAccessToken } from './auth';
 import { ERRORS } from './constants';
 
 const axiosInstance = axios.create();
-// axios instances seem not to have a cancel token, so fixing that here
-axiosInstance.CancelToken = axios.CancelToken;
+let cancelTokenSource = axios.CancelToken.source();
+
+const cancelRequest = () => {
+  cancelTokenSource.cancel();
+  cancelTokenSource = axios.CancelToken.source();
+};
+
+const areRequestsCanceled = err => {
+  return err && axios.isCancel(err);
```

Comment:
is it possible to omit `err &&` part?  `const areRequestsCanceled = err => axios.isCancel(err)` ?